.solhint.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": "solhint:recommended",
+    "plugins": ["prettier"],
+    "rules": {
+        "code-complexity": ["error", 7],
+        "compiler-version": ["error", "^0.8.9"],
```

Comment:
I wish this could be auto detected in some way. I feel like we've had to specify this version in a ton of places :(